src/webserver/WsBase.h

```diff
@@ -452,19 +452,22 @@ class WsSession : public WsBase,
         }(std::move(msg));
 
         boost::json::object request;
-        if (!raw.is_object())
```

Comment:
We shall protect server from any requests, even invalid parameter requests. Bad user can send tons of invalid request to overwhelm server.